### PR TITLE
Updated Add ships

### PR DESCRIPTION
**Pull request by Sunitha101199**

### CODE REVIEW

**Review by VundavalliVaishnavi — COMMENTED** [review]
@Sunitha101199 , Good Job

File: battleship.py [hunk 1]
```diff
@@ -107,8 +107,15 @@ def checkShip(grid, ship):
 Returns: 2D list of ints
 '''
 def addShips(grid, numShips):
```
Comment (VundavalliVaishnavi) [inline battleship.py:107]: @Sunitha101199 , Good Job

**Review by VundavalliVaishnavi — COMMENTED** [review]
@Sunitha101199 , Good Job

**Review by LaxmiNarayanaMurthyVemuri — APPROVED** [review]
Good job!!!!

**Comment by Sunitha101199** [comment]
I have merged this branch addShips with all the four functions to the main.

**Pull request merged**